Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -2314,7 +2314,7 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
 
     void setServiceWorkerOpenWindowCompletionCallback(CompletionHandler<void(std::optional<WebCore::PageIdentifier>)>&&);
 
-#if ENABLE(VIDEO)
+#if ENABLE(VIDEO) && ENABLE(IMAGE_ANALYSIS)
```

Comment:
Did you mean to make this change in https://github.com/WebKit/WebKit/pull/31919?

Reply:
oh, not sure what happened here. Will revert.